src/ui/public/vislib/__tests__/visualizations/column_chart.js

```diff
@@ -58,18 +59,22 @@ dataTypesArray.forEach(function (dataType, i) {
 
       beforeEach(function () {
         vis.handler.charts.forEach(function (chart) {
-          stackedData = chart.stackData(chart.chartData);
+          stackedData = chart.chartData;
 
-          isStacked = stackedData.every(function (arr) {
-            return arr.every(function (d) {
+          isStacked = stackedData.series.every(function (arr) {
+            return arr.values.every(function (d) {
               return _.isNumber(d.y0);
             });
           });
         });
       });
 
       it('should append a d.y0 key to the data object', function () {
-        expect(isStacked).to.be(true);
+        if (mode === 'grouped') {
+          expect(isStacked).to.be(false);
+        } else {
+          expect(isStacked).to.be(true);
```

Comment:
ehm, yeah ... doesnt make much sense ... in the beforeEach i check if d.y0 is there (this means stacking function was run) .... but this test actually tests 'should not stack values if grouped mode is set'